cvat-ui/src/reducers/tasks-reducer.ts

```diff
@@ -238,6 +240,52 @@ export default (state: TasksState = defaultState, action: AnyAction): TasksState
                 },
             };
         }
+        case TasksActionTypes.BACKUP_TASK: {
+            const { taskID } = action.payload;
+            const { backups } = state.activities;
+
+            backups[taskID] = true;
```

Comment:
let's avoid changing current state in this way, in common case we do not know what code refers to the object